homeassistant/components/zoneminder/__init__.py

```diff
@@ -75,7 +76,14 @@ def setup(hass: HomeAssistant, config: ConfigType) -> bool:
         )
         hass.data[DOMAIN][host_name] = zm_client
 
-        success = zm_client.login() and success
+        try:
+            success = zm_client.login() and success
+        except (SSLError, ConnectionError) as ex:
```

Comment:
`requests.exceptions.SSLError` is a subclass of `requests.exceptions.ConnectionError`.

https://github.com/psf/requests/blob/72eccc8dd8b7c272e520f22b0256386c80864e94/src/requests/exceptions.py#L49-L58

```suggestion
        except ConnectionError as ex:
```

Reply:
SSLError is needed to integrate smoothly with https://github.com/Athozs/hass-additional-ca, in which the certs have not been installed yet via that process. This allows the connection to reestablish correctly after the certs have been added from an upgrade.

Reply:
I've adjusted this as 

`except RequestsConnectionError as ex:`